server/routes/topic_page.py

```diff
@@ -55,5 +72,22 @@ def topic_page(topic_id=None, place_dcid=None):
         place_type=place_type,
         place_name=place_name,
         place_dcid=place_dcid,
-        topic_name=topic_place_config.metadata.topic_name,
-        config=MessageToJson(topic_place_config))
+        topic_id=topic_id,
+        topic_name=topic_place_config.metadata.topic_name or "",
+        config=MessageToJson(topic_place_config),
+        topics_summary=topics_summary)
+
+
+def get_topics_summary():
+    topic_place_map = {}
+    topic_name_map = {}
+    all_configs = current_app.config['TOPIC_PAGE_CONFIG']
+    for topic, config_list in all_configs.items():
```

Comment:
nit: topic_id for clarity

```suggestion
    for topic_id, config_list in all_configs.items():
```

Reply:
done